ui/components/app/modular-confirmations/confirmation-row.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Tooltip from '../../ui/tooltip/tooltip';
+import { Box, Icon, IconName, Text } from '../../component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BorderRadius,
+  Color,
+  Display,
+  FlexDirection,
+  FlexWrap,
+  JustifyContent,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+export type ConfirmationRowProps = {
+  label: string;
+  children: React.ReactNode | string;
+  tooltip?: string;
+  variant?: 'default' | 'error' | 'warning';
+};
+
+const BACKGROUND_COLORS = {
+  default: undefined,
+  error: BackgroundColor.errorMuted,
+  warning: BackgroundColor.warningMuted,
+};
+
+const TEXT_COLORS = {
+  default: TextColor.textAlternative,
+  error: Color.errorAlternative,
+  warning: Color.warningAlternative,
+};
+
+const TOOLTIP_ICONS = {
+  default: IconName.Question,
+  error: IconName.Warning,
```

Comment:
[double-checking] did we want to use the warning icon for the error state?

Reply:
I do not recall, maybe @eriknson does?

Reply:
It's the correct icon, but we should call this state `critical` rather than `error` as it'll not (only) be used for displaying errors